kiva/agg/src/graphics_context.i

```diff
@@ -362,12 +362,8 @@ namespace kiva {
                 # Apply base scale for a HiDPI context
                 _agg.GraphicsContextArray_scale_ctm(obj, base_pixel_scale, base_pixel_scale)
 
-                _swig_setattr(self, GraphicsContextArray, 'this', obj)
-                # swig 1.3.28 does not have real thisown, thisown is mapped
-                # to this.own() but with previous 'self.this=obj' an
-                # attribute 'own' error is raised. Does this workaround
-                # work with pre-1.3.28 swig?
-                _swig_setattr(self, GraphicsContextArray, 'thisown2', 1)
```

Comment:
Getting rid of this comment makes it a bit confusing as to why this class is using `thisown2` instead of just `thisown`

Reply:
Unless you're planning to fix the other `thisown2` usages, you shouldn't remove this comment.